src/Acuminator/Acuminator.Analyzers/StaticAnalysis/ThrowingExceptions/WalkerBase.cs

```diff
@@ -40,5 +39,22 @@ protected bool IsPXSetupNotEnteredException(ThrowStatementSyntax node)
             var isSetupNotEntered = exceptionType.InheritsFromOrEquals(_pxContext.Exceptions.PXSetupNotEnteredException);
             return isSetupNotEntered;
         }
+
+        private ITypeSymbol GetExceptionType(ExpressionSyntax node)
```

Comment:
@metallmonster I think we should refactor out and share a part of this method's logic with a switch and place it in Semantic Utils. It could be an extension method, like `GetSymbolType` (with a possibility of further extension)

Reply:
@SENya1990 I was able to achieve the same logic without `GetExceptionType` method: I used `semanticModel.GetTypeInfo(node.Expression)`, it works fine, so I will delete `GetExceptionType` method

Reply:
ok